Demo/src/main/java/com/paypal/android/api/services/PayPalApi.kt

```diff
@@ -2,15 +2,22 @@ package com.paypal.android.api.services
 
 import com.google.gson.JsonObject
 import com.paypal.android.api.model.Order
+import com.paypal.android.data.vault.VaultSessionId
 import retrofit2.http.Body
 import retrofit2.http.Headers
 import retrofit2.http.POST
 
-interface OrdersV2Api {
+interface PayPalApi {
```

Comment:
`PaymentTokensAPI`? because this is what it does

Reply:
it also has `postOrderCheckout` which is part of the `ordersV2` api. I thought of it this way: we have the `paypal` endpoints that will go under here (until it gets too big), and we have the `sample_merchant_server` endpoints which go on the other interface.

Reply:
I could see us reaching a point where we start to migrate interaction with these endpoints into the SDK to support new features.